frontend_lib/src/component/RefreshWarningMessage/RefreshWarningMessage.styl

```diff
@@ -2,15 +2,24 @@
 
 .refreshWarningMessage
   color darkGrey
+  padding 5px 10px
+  background-color lightyellow
+  border 1px solid warningOrange
+  border-radius 5px
+  margin-right 5px
   display flex
-  justify-content flex-end
-  margin 10px 30px 0px
-  white-space nowrap
+  cursor pointer
   .fa
-    margin auto 5px
-  &__button
-    border none
-    background-color transparent
-    color darkGrey
+    margin auto
```

Comment:
do notice that `margin-top: auto` and `margin-bottom: auto` doesn't work, it set margin-top and bottom to 0.
It probably isn't an issue in your case though

Reply:
The value `auto`, set the margin at an "suitable" value, then use `margin-top: auto` and `margin-bottom: auto` is a way to center an element vertically. Using this PR example, here are the screenshots without `margin: auto` and with.
![Screenshot from 2020-06-30 19-34-44](https://user-images.githubusercontent.com/35422692/86158912-cc9a8580-bb09-11ea-8564-16bae354ff1f.png) ![Screenshot from 2020-06-30 19-34-49](https://user-images.githubusercontent.com/35422692/86158916-cdcbb280-bb09-11ea-9867-eaac9565a506.png)
(Maybe it's not super clear in the screenshots, but if you look at the icon compared to the top of the letter R you'll see that one is centralized and the other is not)

Reply:
I have tested it and it indeed seems to do what is expected

Reply:
keep in mind that using margin auto to center vertically might not be the most reliable way. I'm going to look a bit into why it works because I'm a bit confused here ^^

Reply:
I confirm that margin-top auto and margin-bottom auto are interpreted as 0 (see https://developer.mozilla.org/en-US/docs/Web/CSS/margin part "more example")

But I don't see how it works on your branch

Reply:
I just realized the parent of the icon (`.refreshWarningMessage`) is `display: flex`. I'd advise you to give it the property `align-items: center;` and remove margin-top auto and margin-bottom auto

It'll be cleaner

Reply:
I looked at it too and you're right, but the behavior of `margin: auto` changes when the parent element has `display: flex`. 
See: https://css-tricks.com/the-peculiar-magic-of-flexbox-and-auto-margins/

Reply:
mmh
nice catch, today I learned

Reply:
Cool :smile:. Anyway, I decided to switch to something less "magical".